docs/release-notes.md

```diff
@@ -10,6 +10,68 @@ toc_max_heading_level: 2
 
 Welcome to Semgrep release notes. This document provides an overview of the changes, additions, and selected important fixes. Release notes published since April 2022 include Semgrep CLI, CI, and Semgrep App updates. Release notes published since April 2022 also include updates from more versions collected together.
 
+## July 2022
+
+### Semgrep CLI and Semgrep in CI
```

Comment:
We shouldn’t use “Semgrep in CI” as a product name in the similar way to how we used to use “Semgrep CI.”

Reply:
What do you suggest, only Semgrep CLI here for the header? @pabloest

Reply:
Resolved in: https://github.com/returntocorp/semgrep-docs/pull/640#discussion_r933663338